service/history/replicatorQueueProcessor_mock.go

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
```

Comment:
plz use mockgen, e.g. `mockgen -source ./service/history/nDCWorkflow.go -destination ./service/history/nDCTransactionMgrForNewWorkflow_mock.go -package history`

this mock lib provides better testing functionality

Reply:
done